### PR TITLE
ES6 toms quiz

### PR DESCRIPTION
**Pull request by regiskuckaertz**

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/atoms/quiz.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
+// @flow
+import bean from 'bean';
+import fastdom from 'lib/fastdom-promise';
+
+export default {
```
Comment (SiAdcock) [inline static/src/javascripts/projects/common/modules/atoms/quiz.js]: Is there a way we could only export `handleCompletion` as a standalone function?

Comment (regiskuckaertz) [inline static/src/javascripts/projects/common/modules/atoms/quiz.js]: what do you mean? I thought you did not want that ❓

Comment (SiAdcock) [inline static/src/javascripts/projects/common/modules/atoms/quiz.js]: Sorry for my lack of clarity... you have correctly deciphered my suggestion!

**Review by SiAdcock — APPROVED** [review]
Great!

**Comment by regiskuckaertz** [comment]
Thanks hombre :cowboy_hat_face:

**Pull request merged**